src/Mod/CAM/Path/Post/scripts/grbl_post.py

```diff
@@ -405,10 +400,9 @@ def export(objectslist, filename, argstring):
     print("Done postprocessing.")
 
     # write the file
-    if not filename == "-":
-        gfile = pythonopen(filename, "w")
-        gfile.write(final)
-        gfile.close()
```

Comment:
I don't particularly mind it but why change this here but not in the rest?

Reply:
I have to admit that I'm not familiar enough with the source to realize that the same pattern appears in other files as well. Please let me know if you want me to revert it and open another pull request which fixes it globally.